e-vent.ino

```diff
@@ -265,6 +266,13 @@ void loop() {
         displ.writePEEP(round(pressureReader.peep()));
         displ.writePlateauP(round(pressureReader.plateau()));
         setState(IN_STATE);
+
+        // An indicator that shows patient-triggered or time-triggered cycles
+        if(patientTriggered) {
+          displ.showPatientIcon(3,11);
```

Comment:
Although this seems to be fine right now, it's fragile because `writePEEP` and this are writing to the same place in the display, so there's an unnecessary implied requirement for this to be called right after `writePEEP`; not to mention the wasted write. We can discuss this more offline.